### ISSUE TITLE
Remove Mapbox Classic as Other Basemap Option

### ISSUE THREAD

**Issue by jasonash** [StraboSpot/strabo-mobile#534]
Doug has requested that we remove Mapbox Classic as an Other Basemap option due to its deprecation at Mapbox.

**Comment by NathanNovak** [comment]
@jasonash ok sounds good.  Did you want it removed for this version?

**Comment by jasonash** [comment]
@NathanNovak No, nothing can be changed for this version. It takes around three days for the app store to approve things, so we're locked in now. I will take care of this one for the next release.

**Comment by NathanNovak** [comment]
ok
